rkt.spec

```diff
@@ -97,10 +120,24 @@ install -dp -m 700 %{buildroot}%{_sharedstatedir}/%{name}/containers
 %{_unitdir}/%{name}-metadata.socket
 %{_unitdir}/%{name}-metadata.service
 
-%{_sharedstatedir}/%{name}/cas
-%{_sharedstatedir}/%{name}/cas/db
-%{_sharedstatedir}/%{name}/tmp
-%{_sharedstatedir}/%{name}/containers
+%attr(2770, root, rkt) %{_sharedstatedir}/%{name}
```

Comment:
Yes, just `rkt install`  without arguments.

I was wondering if not setting the perms in the spec would mess with audit tools that could use `rpm --verify` and notice the difference. But if it is not a problem, then all good.

https://github.com/coreos/rkt/blob/master/Documentation/packaging.md#ownership-and-permissions-of-rkt-directories